src/onboarding/welcome/Welcome.tsx

```diff
@@ -53,7 +54,15 @@ export default function Welcome() {
       <Image source={welcomeBackground} style={styles.backgroundImage} />
       <ScrollView contentContainerStyle={styles.contentContainer}>
         <Logo type={LogoTypes.COLOR} height={64} />
-        <Text style={styles.title}>{t('welcome.title')}</Text>
+        {showGuidedOnboarding ? (
+          <Text style={styles.title} testID={'WelcomeText'}>
+            {t('welcome.titleGuided')}
+          </Text>
+        ) : (
+          <Text style={styles.title} testID={'WelcomeText'}>
+            {t('welcome.title')}
+          </Text>
+        )}
```

Comment:
```suggestion
          <Text style={styles.title} testID={'WelcomeText'}>
            {t(showGuidedOnboarding ? 'welcome.titleGuided' : 'welcome.title')}
          </Text>
```
code golf

Reply:
code formatting's probly wrong but you get the idea

Reply:
Actually surprised that I wrote it the way I did; will send a commit and fix it.